sapi/cli/tests/php_cli_server_pdeathsig.phpt

```diff
@@ -35,7 +35,7 @@ if (count($workersBefore) === 0) {
 
 proc_terminate($cliServerInfo->processHandle, 9); // SIGKILL
 
-usleep(10000);
+sleep(1);
```

Comment:
this is huge step - from 10ms to 1s

it seems there is some timer with limited precision used, which should be probably fixed instead

Reply:
I think the prolem is: https://github.com/php/php-src/blob/3227d04fa9055f488571efbe6e30a72ba9ac2b2c/ext/standard/basic_functions.c#L1185

see my old closed PR - https://github.com/php/php-src/pull/5961 - we should fallback to other available sleep function

Reply:
@mvorisek The test is new. I added the sleep and it probably just succeeded by chance. Some form of "polling" would probably be best (checking and then sleeping for a short amount of time, for a limited number of iterations).

Reply:
if new... :)

Reply:
Improved it on `master` directly.